Game of Survivor

PavelEA opened the pull request:
add pom.xml

PavelEA committed:
First commit

PavelEA committed:
Add styles and pictures

PavelEA committed:
Restored and added

PavelEA committed:
Restored and added style.css

PavelEA committed:
Creating utility classes

PavelEA committed:
Adding Servlets and Fixing pom.xml

PavelEA committed:
Moving styles, files jsp, pictures. Optimized import.

PavelEA committed:
Optimized import and fixing errors after decompilation

akupreychik commented on src/main/java/dev/PavelEA/GoS/servlet/AppContextListener.java:
```diff
@@ -0,0 +1,20 @@
+public class AppContextListener implements ServletContextListener {
+    @Override
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
+        ServletContext ctx = servletContextEvent.getServletContext();
```
не нужно сокращать, лучше использовать полные названия

akupreychik reviewed (commented):
не хватает тестов + мне кажется очень много лишних jsp страниц, все данные можно было передавать при инициализации страницы + параметры которые ты бы хотел получить

akupreychik commented:
но все получилось отлично! ничего лишнего) хорошая работа)

PavelEA committed:
Create SessionFactoryProvider, connect to Hibernate

PavelEA committed:
Create new class UserRepositoryDB and create some method(save,update,find).
Add annotation on entity User

PavelEA committed:
Dependencies are fixed, Hibernate is connected, the work of the save method with adding an entity to the database is implemented

PavelEA committed:
1.In RegistrationServlet, the logic of saving and deleting users is fully implemented. Partially implemented work list of users.
2. In the settingGame and startPage, extra lines have been removed, the code has been formatted.
3. StoryOne updated import
4. In the file with styles, the specific path for the image for the background has been removed (required for use in the header)
5. Methods are fully implemented in UserRepositoryDB: usernameExists, getAllUserId, getAllUserUsername.
Operation with transaction has been added to the deleteByUsername method.
6. villageSob24 fixed a bug with the lack of content display.
